src/Compilers/Core/Portable/Syntax/LineDirectiveMap.cs

```diff
@@ -19,7 +20,7 @@ namespace Microsoft.CodeAnalysis
     internal abstract partial class LineDirectiveMap<TDirective>
         where TDirective : SyntaxNode
     {
-        protected readonly LineMappingEntry[] Entries;
+        protected readonly ImmutableArray<LineMappingEntry> Entries;
```

Comment:
I suggest you to only keep the changes that affect behavior in the way you want it to change. Changing the type of this field doesn't appear to be related to any desired behavior change it can be done post RTM.